src/content/configuration/output.md

```diff
@@ -356,86 +350,111 @@ MySubModule.doSomething();
 
 ```javascript
 this["MyLibrary"] = _entry_return_;
 
-// your users will use your library like:
+// your users will use your library like the following:
 this.MyLibrary.doSomething();
 MyLibrary.doSomething(); //if this is window
 ```
 
 
-`libraryTarget: "window"` - When your library is loaded, **the return value of your entry point** will be part `window` object.
+`libraryTarget: "window"` - When your library is loaded, **the return value of your entry point** will be part `window` object under the property named by `output.library`.
 
  ```javascript
  window["MyLibrary"] = _entry_return_;
 
-//your users will use your library like:
+//your users will use your library like the following:
 window.MyLibrary.doSomething();
  ```
 
 
-`libraryTarget: "global"` - When your library is loaded, **the return value of your entry point** will be part `global` object.
+`libraryTarget: "global"` - When your library is loaded, **the return value of your entry point** will be part `global` object under the property named by `output.library`.
 
  ```javascript
  global["MyLibrary"] = _entry_return_;
 
-//your users will use your library like:
+//your users will use your library like the following:
 global.MyLibrary.doSomething();
  ```
 
 
-`libraryTarget: "commonjs"` - When your library is loaded, **the return value of your entry point** will be part of the exports object. As the name implies, this is used in CommonJS environments:
+`libraryTarget: "commonjs"` - When your library is loaded, **the return value of your entry point** will be part of the `exports` object under the property named by `output.library`. As the name implies, this is used in CommonJS environments:
 
 ```javascript
 exports["MyLibrary"] = _entry_return_;
 
-//your users will use your library like:
+//your users will use your library like the following:
 require("MyLibrary").doSomething();
 ```
 
+### Exposing the library under formalized module definition systems
+
+These options will result in a bundle that comes with a more complete header to ensures compatibility with various module systems.  The `output.library` option will take on a different meaning under the following `output.libraryTarget` options.
+
 
 `libraryTarget: "commonjs2"` - When your library is loaded, **the return value of your entry point** will be part of the exports object. As the name implies, this is used in CommonJS environments:
 
 ```javascript
 module.exports = _entry_return_;
 
-//your users will use your library like:
+//your users will use your library like the following:
 require("MyLibrary").doSomething();
 ```
 
-T> Wondering the difference between CommonJS and CommonJS2? Check [this](https://github.com/webpack/webpack/issues/1114) out (they are pretty much the same).
-
+Note that `output.library` is omitted, thus it is not required for this particular `output.libraryType`.
```

Comment:
Again unless I've missed something and `libraryType` is a configuration option, I think we should rephrase the last bit as "... this particular library type."